### PR TITLE
Prepare Storage for release

### PR DESCRIPTION
**Pull request by seanmcc-msft**

### CODE REVIEW

File: sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Release History
 
-## 5.0.0-beta.1 (Unreleased)
+## 5.0.0-beta.1 (2020-11-10)
+- First release.
```
Comment (amnguye) [inline sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/CHANGELOG.md:4]: NIT - Should this have the same description as the other WebJobs Changelogs?

**Review by amnguye — APPROVED** [review]

File: sdk/storage/Azure.Storage.Common/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 12.6.0-preview.2 (Unreleased)
-
+## 12.6.0 (2020-11-10)
+- This release contains bug fixes to improve quality.
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Common/CHANGELOG.md:4]: Weren't there new SAS features added to Common?

File: sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 5.0.0-beta.1 (Unreleased)
+## 5.0.0-beta.1 (2020-11-10)
 
 This is the first preview of the next generation of `Microsoft.Azure.WebJobs.Extension.Storage` which has been integrated with latest Azure Storage SDK that follows the [Azure SDK Design Guidelines for .NET](https://azure.github.io/azure-sdk/dotnet_introduction.html) which provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more.
```
Comment (tg-msft) [inline sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/CHANGELOG.md:5]: Should this say `.Storage.Queues`?

File: sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/CHANGELOG.md (hunk 1 above)
Comment (kasobol-msft) [reply sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/CHANGELOG.md:4]: Good idea, but let's just use this:
```
This is the first preview of the next generation of `Microsoft.Azure.WebJobs.Extension.Storage` which has been integrated with latest Azure Storage SDK that follows the [Azure SDK Design Guidelines for .NET](https://azure.github.io/azure-sdk/dotnet_introduction.html) which provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more.
```

File: sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 5.0.0-beta.1 (Unreleased)
+## 5.0.0-beta.1 (2020-11-10)
 
 This is the first preview of the next generation of `Microsoft.Azure.WebJobs.Extension.Storage` which has been integrated with latest Azure Storage SDK that follows the [Azure SDK Design Guidelines for .NET](https://azure.github.io/azure-sdk/dotnet_introduction.html) which provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more.
```
Comment (tg-msft) [inline sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/CHANGELOG.md:5]: Should this say `.Storage.Blobs`?

**Review by kasobol-msft — APPROVED** [review]

File: sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/CHANGELOG.md (hunk 4 above)
Comment (kasobol-msft) [reply sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/CHANGELOG.md:5]: the sentence refers to the past where `Microsoft.Azure.WebJobs.Extension.Storage` wasn't split.

File: sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/CHANGELOG.md (hunk 3 above)
Comment (kasobol-msft) [reply sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/CHANGELOG.md:5]: the sentence refers to the past where `Microsoft.Azure.WebJobs.Extension.Storage` wasn't split.

Comment (kasobol-msft) [reply sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/CHANGELOG.md:5]: The changelog mentions the split further down.

File: sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/CHANGELOG.md (hunk 1 above)
Comment (kasobol-msft) [reply sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/CHANGELOG.md:4]: for reference: this package doesn't go to nuget today. it's all shared source.

Comment (seanmcc-msft) [reply sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/CHANGELOG.md:4]: fixed.

File: sdk/storage/Azure.Storage.Common/CHANGELOG.md (hunk 2 above)
Comment (amnguye) [reply sdk/storage/Azure.Storage.Common/CHANGELOG.md:4]: The only new thing added to Common was a new constructor for `AccountSasBuilder`, so `AccountSasBuilder(permissions, expiresON, services, resourceTypes)`. It was more for ease of the customer knowing what's the minimum requirements to create a SAS and to construct the SASs from the Service Clients. It's not really a new feature.

**Pull request merged**